### PR TITLE
[Docathon][Add Overview Doc No.14] add doc of docathon 14

### PR DESCRIPTION
**Pull request by Wang-ZengWei**
14. 新增Overview
      [paddle.distributed.ParallelEnv](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/distributed/ParallelEnv_cn.html)
      [paddle.distributed.split](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/distributed/split_cn.html)
      [paddle.distributed.wait](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/distributed/wait_cn.html)
      [paddle.distributed.fleet.utils.recompute](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/distributed/fleet/utils/recompute_cn.html)
      
中文文档链接：[#6427](https://github.com/PaddlePaddle/docs/issues/6427)
@sunzhongkai588 @Turingg

### CODE REVIEW

File: docs/api/paddle/distributed/Overview_cn.rst [hunk 1]
```diff
@@ -100,6 +102,7 @@ Fleet 分布式高层 API
     " :ref:`send <cn_api_paddle_distributed_send>` ", "发送一个 tensor 到指定进程"
     " :ref:`recv <cn_api_paddle_distributed_recv>` ", "接收一个来自指定进程的 tensor"
     " :ref:`barrier <cn_api_paddle_distributed_barrier>` ", "同步路障，阻塞操作以实现组内进程同步"
+    " :ref:`wait <cn_api_paddle_distributed_wait>` ", "同步通信组"
```
Comment (Turingg) [inline docs/api/paddle/distributed/Overview_cn.rst]: ”同步通信组，在指定的通信组中同步特定的 tensor 对象“ 是否会更好一些呢

**Comment by Wang-ZengWei** [comment]
reopen

**Comment by Wang-ZengWei** [comment]
reopen

**Comment by Wang-ZengWei** [comment]
1

**Comment by Wang-ZengWei** [comment]
reopen

**Review by sunzhongkai588 — APPROVED** [review]
LGTM~

**Pull request merged**